src/css/modules/pages/home.css

```diff
@@ -38,10 +38,15 @@
 
 @media (min-width: 50em) {
   .home-model::before {
-    background-image: url("/images/content-connector-line.svg");
+    background-image: url('/images/content-connector-line.svg');
     background-position: top left;
   }
 }
+@media (max-width: 800px) {
```

Comment:
@will0101 I notice these CSS changes are in another PR. Do these need to be here too or can we remove them here and only have the broken link update?

Reply:
These can be removed.

Reply:
Cool. Do you know why they made it here? Was this a branch off your other feature branch instead of `main`?

Reply:
That's exactly what happened. I'll watch out for this mistake next time.